tests/scripts/run-throughput-perf.py

```diff
@@ -166,12 +173,12 @@ def validate_arg(arg, check):
         if not helper(arg):
             raise Exception('Argument: %s is not valid.' % (arg))
 
-    valid_archs = {'Windows_NT': ['x86', 'x64'], 'Linux': ['x64']}
+    valid_archs = {'Windows_NT': ['x86', 'x64'], 'Linux': ['x64', 'arm']}
     valid_build_types = ['Release']
     valid_run_types = ['rolling', 'private']
     valid_os = ['Windows_NT', 'Ubuntu14.04', 'Ubuntu16.04']
     valid_opt_levels = ['full_opt', 'min_opt']
-    valid_jit_names = {'x64': ['ryujit'], 'x86': ['ryujit', 'legacy_backend']}
+    valid_jit_names = {'x64': ['ryujit'], 'x86': ['ryujit', 'legacy_backend'], 'arm': ['ryujit', 'legacy_backend']}
```

Comment:
I believe legacy_backend can safely be removed @BruceForstall to confirm

Reply:
I believe all references to `legacy_backend` can be safely removed.

Reply:
Cool. Removed.